src/formatError.spec.ts

```diff
@@ -0,0 +1,19 @@
+import test from 'ava'
+import {AssertOrder} from './assertOrder'
+
+test('Testing message for runOnce error case', _t => {
+  let a = new AssertOrder();
+  _t.throws(() => a.runOnce(1), "Expecting 'runOnce(0)', 'step(0)', 'runAtleastOnce(0)', 'repeatExactCount(0)', 'multiple(0)', but received 'runOnce(1)'")
```

Comment:
Would be best if we can test the `formatError()` directly.
When these tests passed, then we can return the `formatError()` code in `AssertOrder.spec.ts` to vastly simplifies the tests.